samples/openapi3/client/petstore/ruby/docs/FakeApi.md

```diff
@@ -531,11 +531,11 @@ end
 
 api_instance = Petstore::FakeApi.new
 required_string_group = 56 # Integer | Required String in group parameters
-required_boolean_group = true # BOOLEAN | Required Boolean in group parameters
+required_boolean_group = nil # Boolean | Required Boolean in group parameters
```

Comment:
@autopp I'll file another PR to use true/false instead of nil for boolean example value